.formatter.exs

```diff
@@ -76,7 +76,17 @@ locals_without_parens = [
   on_mount: 1,
   slot: 1,
   slot: 2,
-  slot: 3
+  slot: 3,
+
+  # Phoenix.LiveViewTest
+  assert_patch: 2,
```

Comment:
Do we also need `assert_patch/1` and `assert_redirect/1`? (@josevalim)

The second parameter can be an optional timeout:

https://hexdocs.pm/phoenix_live_view/Phoenix.LiveViewTest.html#assert_patch/2
https://hexdocs.pm/phoenix_live_view/Phoenix.LiveViewTest.html#assert_redirect/2

```elixir
assert_patch view
```

(the same might apply with assert_push_event/3 so if we need the ones above, please double check)

Reply:
Yep, we need to put them there, doing it!